src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -1,12 +1,30 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
-from . import ApiController, AuthRequired, RESTController
+import json
+import cherrypy
+
+from mgr_module import CommandResult
+from . import ApiController, AuthRequired, RESTController, Endpoint
 from .. import mgr
+from ..services.ceph_service import CephService
 
 
 @ApiController('/host')
 @AuthRequired()
 class Host(RESTController):
     def list(self):
         return mgr.list_servers()
+
+    @Endpoint('PUT')
+    def scrub(self, hosts, deep_scrub):
```

Comment:
I think this operation belongs to the OSD controller, or should go into a new controller, because when setting the security scope and permissions for this operation (PR #22283) we need this operation to be inside a controller belonging to the `Scope.OSD`, which is not currently the case.